README.md

```diff
@@ -130,6 +130,19 @@ These values allow you to configure the generator.
 * `initialCommitOptions`: Array of additional git options to be passed to the initial commit.
 * `storeResponses`: If this is set to `true`, create-using-template will store any responses for your template for future template generations.
 
+##### `lifecycle` object<function>, optional
+
+These functions will run at specific stages of the generation lifecycle<!-- .slide: data-fullscreen -->
```

Comment:
What does `<!-- .slide: data-fullscreen -->` do?

Reply:
I have no idea where that comment came from. I didn't type that.

Reply:
removed